tests/unit/customizations/s3/test_subcommands.py

```diff
@@ -132,6 +132,26 @@ def test_ls_command_with_no_args(self):
             's3', region_name=None, endpoint_url=None, verify=None,
             config=None))
 
+    def test_ls_command_with_non_existsent_url(self):
+        ls_command = ListCommand(self.session)
+        parsed_global = FakeArgs(region=None, endpoint_url=None,
```

Comment:
Can we fix the indentation of the following line to line up with this line? Similar to https://github.com/aws/aws-cli/blob/develop/tests/unit/customizations/s3/test_subcommands.py#L114-L115